symphony/bdk/core/config/model/bdk_certificate_config.py

```diff
@@ -21,3 +21,20 @@ def is_valid(self) -> bool:
         :return: true if the certificate configuration is valid.
         """
         return not (self.path is not None and self.content != "")
+
+    def set_content(self, certificate_content):
+        """Sets ceritficate content and overrides path to None to keep certificate config valid
+        Args:
```

Comment:
The markup used for the python BDK documentation is reStructuredText. 
Here is my [cheat sheet](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html), could be helpful.
This means you should use:
```python
def function(param1):
''' Some description

:param param1 description of parameter
'''
```